app/components/createContainer/createContainerController.js

```diff
@@ -13,13 +13,24 @@ function ($q, $scope, $state, $transition$, $filter, Container, ContainerHelper,
     ExtraHosts: [],
     IPv4: '',
     IPv6: '',
-    AccessControlData: new AccessControlFormData()
+    AccessControlData: new AccessControlFormData(),
+    CpuLimit: 0,
+    MemoryLimit: 0,
+    MemoryReservation: 0,
+    MemoryLimitUnit: 'MB',
```

Comment:
Is there any use for the `MemoryLimitUnit` and `MemoryReservationUnit` vars as we now enforce it to MB ?